readme.md

```diff
@@ -159,13 +168,13 @@ Example:
 import {expectType} from 'tsd';
 import delay from '.';
 
-expectType<void>(await delay(200));
+expectType<Promise<void>>(delay(200));
 
-expectType<string>(await delay(200, {value: '🦄'}));
-expectType<number>(await delay(200, {value: 0}));
+expectType<Promise<string>>(delay(200, {value: '🦄'}));
+expectType<Promise<number>>(delay(200, {value: 0}));
 
-expectType<never>(await delay.reject(200, {value: '🦄'}));
-expectType<never>(await delay.reject(200, {value: 0}));
+expectType<Promise<never>>(delay.reject(200, {value: '🦄'}));
+expectType<Promise<never>>(delay.reject(200, {value: 0}));
```

Comment:
Can you add a note about not using `await` and why? I don't think that's obvious to people even if they see Promise used here. It was at least not obvious to me.